config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml

```diff
@@ -55,6 +55,347 @@ spec:
                 items:
                   type: string
                 type: array
+              affinity:
```

Comment:
Is this the same schema as for pod affinity rules? Can we just link to that? If we're just passing it through is there anyway to specify that in the yaml and explicitly track that API?

Reply:
Don’t quite get what you are saying

Reply:
I don't think it's possible but just saying
```
affinity: *podSpecAffinitySchema
```
Linking to that block of yaml rather than copy pasting the whole thing

Reply:
We are using the strict from K8s in the api